src/test/ui/macros/auxiliary/issue-75982.rs

```diff
@@ -0,0 +1,4 @@
+const _: () = {
+    #[path = "issue-75982-foo.rs"]
+    mod foo;
```

Comment:
Should I change this, or leave the test the same?

Reply:
I'd remove the `path` attribute and the file `issue-75982-foo.rs` at least.
Also, it would be nice to have two macros - one directly in the const block, and one inside `mod foo` to test both cases.